src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -9,7 +9,6 @@
 import org.testng.annotations.Test;
 
 import teammates.common.util.Const;
-import teammates.common.util.FieldValidator;
```

Comment:
Remove lines 3-5 instead and let this import be. You'll also need to add `FieldValidator.` to all the method calls in this file.